LocalPackages/NetworkProtectionMac/Sources/NetworkProtectionUI/Views/TunnelControllerView/TunnelControllerViewModel.swift

```diff
@@ -68,25 +78,13 @@ public final class TunnelControllerViewModel: ObservableObject {
 
     private let uiActionHandler: VPNUIActionHandling
 
-    // MARK: - Environment
-
-    @EnvironmentObject
-    private var siteTroubleshootingViewModel: SiteTroubleshootingView.Model
-
     // MARK: - Misc
 
     /// The `RunLoop` for the timer.
     ///
     private let runLoopMode: RunLoop.Mode?
     private var cancellables = Set<AnyCancellable>()
 
-    // MARK: - Dispatch Queues
-
-    private static let statusDispatchQueue = DispatchQueue(label: "com.duckduckgo.NetworkProtectionStatusView.statusDispatchQueue", qos: .userInteractive)
-    private static let connectivityIssuesDispatchQueue = DispatchQueue(label: "com.duckduckgo.NetworkProtectionStatusView.connectivityIssuesDispatchQueue", qos: .userInteractive)
-    private static let serverInfoDispatchQueue = DispatchQueue(label: "com.duckduckgo.NetworkProtectionStatusView.serverInfoDispatchQueue", qos: .userInteractive)
-    private static let dataVolumeDispatchQueue = DispatchQueue(label: "com.duckduckgo.NetworkProtectionStatusView.dataVolumeDispatchQueue", qos: .userInteractive)
```

Comment:
Most of these are for refreshing the UI so I moved everything to the main queue.